src/System.Private.ServiceModel/tools/SelfHostedWcfService/SelfHostedWCFService.csproj

```diff
@@ -60,6 +60,9 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="..\IISHostedWcfService\Web.config">
+      <Link>Web.config</Link>
+    </None>
```

Comment:
It's a pain when adding a new endpoint to go find the web.config file to add the endpoint there as well as in Program.cs. This just adds the file to the project so it's easier to edit. It doesn't get copied and nothing is done with it at all when compiling, it's just easier to edit having it right there.